packages/react-components/src/AccessibilityScenarios/ToggleButton.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { ToggleButton } from '@fluentui/react-button';
+
+import { Scenario } from './utils';
+
+export const DeviceControlsToggleButtonsAccessibilityScenario: React.FunctionComponent = () => {
+  const [hotspotButtonDisabled, setHotspotButtonDisabled] = React.useState<boolean | undefined>(true);
+  const [wiFiButtonPressed, setWiFiButtonPressed] = React.useState(false);
+  const [hotspotButtonPressed, setHotspotButtonPressed] = React.useState(false);
+
+  const onWiFiButtonClick = (event: React.MouseEvent) => {
+    setHotspotButtonDisabled(hotspotButtonDisabled ? undefined : true);
+    if (wiFiButtonPressed) {
+      setHotspotButtonPressed(false);
+    }
+    setWiFiButtonPressed(!wiFiButtonPressed);
+  };
+
+  const onHotspotButtonClick = () => {
+    setHotspotButtonPressed(!hotspotButtonPressed);
+  };
+
+  return (
+    <Scenario pageTitle="Device controls toggle buttons">
+      <ToggleButton onClick={onWiFiButtonClick}>Wi-Fi</ToggleButton>
```

Comment:
I think it'd be good to add a heading and maybe a short line of descriptive text to give users a little bit of context about what the buttons are for.

Reply:
Okay, added. Please, check it again.